src/components/Movies/MovieDescription.tsx

```diff
@@ -1,25 +1,48 @@
 import React, { FC, useMemo } from 'react'
 
 import styles from './MovieDescription.scss'
-import { Movie } from "./MovieCard";
-import {faSearch} from "@fortawesome/free-solid-svg-icons";
-import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import { faSearch } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { useGetMoviesQuery } from '../../app/moviesApi'
+import { useDispatch, useSelector } from 'react-redux'
+import { RootState } from '../../app/store'
+import { removeSelectedMovie } from '../../features/movies/moviesSlice'
 
-interface MovieDescriptionProps extends Movie {
-  onClose: () => void
-}
+const MovieDescription: FC = () => {
+  const dispatch = useDispatch()
+  const sortOrder = useSelector((state: RootState) => state.movies.sortOrder)
```

Comment:
I think you can do something like 

`const { sortOrder, sortBy, genreSelected, selectedMovie } = useSelector((state: RootState) => state.movies)`

Same for other places.